Cargo.toml

```diff
@@ -359,7 +359,7 @@ bench = false
 # To use a development version of a dependency please use a global override here
 # changing versions in each sub-crate of the workspace is tedious
 [patch.crates-io]
-reedline = { git = "https://github.com/nushell/reedline", branch = "main" }
+reedline = { git = "https://github.com/nushell/reedline", rev = "68084cde80e366fa5787079064bb65b710f23a97" }
```

Comment:
I believe this line needs to be reverted. It just needs to be `branch = "main"` since the cargo.lock is on commit 68084cd in this PR.